package.json

```diff
@@ -47,22 +47,27 @@
     "@angular/platform-browser-dynamic": "13.2.2",
     "@angular/router": "13.2.2",
     "@angular/service-worker": "13.2.2",
+    "@ew-did-registry/credentials-interface": "^0.6.3-alpha.596.0",
```

Comment:
i think this is not needed. Those libs are added by ICL, i don't think we need to care about them here.

Reply:
I think we need to import these to import the types/interfaces